pkgs/development/tools/wails/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+, pkg-config
+, makeWrapper
+, go
+, gcc
+, gtk3
+, webkitgtk
+, nodejs
+, upx
+, zlib
+}:
+
+buildGoModule rec {
+  pname = "wails";
+  version = "2.0.0-beta.33";
+
+  src = fetchFromGitHub
+    {
+      owner = "wailsapp";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "efxOL/FllToum0P4JyAJt0fbrznTFYh7czTWpZu3uk0=";
+    } + "/v2";
```

Comment:
```suggestion
  src = fetchFromGitHub {
    owner = "wailsapp";
    repo = pname;
    rev = "v${version}";
    sha256 = "efxOL/FllToum0P4JyAJt0fbrznTFYh7czTWpZu3uk0=";
  } + "/v2";
```

Reply:
I don't like the dropped brace either, but `nixpkgs-fmt` kept putting it back to that.

But now that I look at it, maybe I had an incorrect indent of the entries or trailing brace and that caused the drop to happen. I'll try and fix this up.

Reply:
`nixpkgs-fmt` only does this if the syntax is wonky. Otherwise I think this is a bug and shouldn't happen.